sympy/stats/drv_types.py

```diff
@@ -114,6 +116,91 @@ def Geometric(name, p):
     return rv(name, GeometricDistribution, p)
 
 
+#-------------------------------------------------------------------------------
+# Hermite distribution ---------------------------------------------------------
+
+
+class HermiteDistribution(SingleDiscreteDistribution):
+    _argnames = ('a1', 'a2')
+    set = S.Naturals0
+
+    @staticmethod
+    def check(a1, a2):
+        _value_check(a1.is_positive or a1.is_zero, 'Parameter a1 must be >= 0.')
+        _value_check(a2.is_positive or a2.is_zero, 'Parameter a2 must be >= 0.')
+
+    def pdf(self, k):
+        a1, a2 = self.a1, self.a2
+        term1 = exp(-(a1 + a2))
+        j = Dummy("j", integer=True)
+        num = a1**(k - 2*j) * a2**j
+        den = factorial(k - 2*j) * factorial(j)
+        return term1 * Sum(num/den, (j, 0, k//2)).doit()
+
+    def _moment_generating_function(self, t):
+        a1, a2 = self.a1, self.a2
+        term1 = a1 * (exp(t) - 1)
+        term2 = a2 * (exp(2*t) - 1)
+        return exp(term1 + term2)
+
+    def _characteristic_function(self, t):
+        a1, a2 = self.a1, self.a2
+        term1 = a1 * (exp(I*t) - 1)
+        term2 = a2 * (exp(2*I*t) - 1)
+        return exp(term1 + term2)
```

Comment:
If we remove these two functions, the results are not that good looking as this, `Sum(num/den, (j, 0, k//2)).doit()` can not represented in closed form but we can remove this pair as IMHO, it's not `stats` moudle's concern to generate simplified as ideally it should have been handled by `Sum`.

Reply:
I tried by removing them, but then, all the results i.e E, kurtosis, moment_generating_function, characteristic_function, skewness are not good as expected.
Removing them I get following results,
```
>>>E(X)
Sum(X*exp(-a1 - a2)*Sum(a1**(-2*_j + X)*a2**_j/(factorial(_j)*factorial(-2*_j + X)), (_j, 0, floor(X/2))), (X, 0, oo))

>>> moment_generating_function(X)(x)
Sum(exp(_x*x)*exp(-a1 - a2)*Sum(a1**(-2*_j + _x)*a2**_j/(factorial(_j)*factorial(-2*_j + _x)), (_j, 0, floor(_x/2))), (_x, 0, oo))

>>> characteristic_function(X)
Sum(exp(_x*I*x)*exp(-a1 - a2)*Sum(a1**(-2*_j + _x)*a2**_j/(factorial(_j)*factorial(-2*_j + _x)), (_j, 0, floor(_x/2))), (_x, 0, oo))

```
While kurtosis and skewness are taking too long to get computed.
So , in my opinion they should be kept added as of for now.

Reply:
Yeah they don't look good. It's okay to keep them though it's a bit ad-hoc.